logger_test.go

```diff
@@ -94,7 +95,7 @@ func TestLogWithFieldsAndStacktrace(t *testing.T) {
 	// test stack trace strings
 	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger.justToShowUpInStackTrace")
 	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger.TestLogWithFieldsAndStacktrace")
-	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger/logger_test.go:122")
+	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger/logger_test.go:124")
```

Comment:
you added one line but this changed by 2, why?

Reply:
good eye, i changed without running the test... actually it was breaking